src/App.tsx

```diff
@@ -61,11 +62,12 @@ function App() {
       <div>
         <GlobalStyle />
         <ApplicationContext.Provider value={application}>
-          <UIPluginSelector
-            absoluteDottedId={`${dataSourceId}/${application?._id}`}
-            type={application?.type}
-            categories={['Application']}
-          />
+          <FSTreeProvider visibleDataSources={application.dataSources}>
```

Comment:
Is treeprovider necessary here? looks like it is also used in src/plugins/analysis-platform/App.tsx.

Maybe we can remove treeprovider from src/plugins/analysis-platform/App.tsx.

Reply:
That file will be removed or "plugified". Not in use now.
Ether way, these context providers should be in the top level "App.tsx", as that is provided by "create-dm-app".

Reply:
Removed it from "analysis-platform/App.tsx" :+1: